packages/fe/components/cid-card.vue

```diff
@@ -224,6 +252,11 @@ export default {
       required: true,
       default: () => ({})
     },
+    copy: {
```

Comment:
Let's rename this to `content` to match the `content` directory's name. `copy` is confusing because this same component has a "copy button" that does copy/pasting.

Also, can you rename all the keys inside this content object more to be more descriptive + human-readable? For example:

```
copy.status → copy.statusLabel
copy.storageProviders → copy.storageProvidersHeading
```

The reason for this is to make it extra clear when reading the code that a label is being outputted and not something else. For example, `copy.status` could output something else like "Pending" rather than a label. And `copy.storageProviders` sounds like it outputs a list of storage providers or possibly a sentence noting several storage providers, rather than a heading.